apps/cowswap-frontend/src/modules/appData/updater/AppDataInfoUpdater.ts

```diff
@@ -3,15 +3,16 @@ import { useEffect, useRef } from 'react'
 
 import { CowEnv, SupportedChainId } from '@cowprotocol/cow-sdk'
 
+import { AppCodeWithWidgetMetadata } from 'modules/injectedWidget/hooks/useAppCodeWithWidgetMetadata'
 import { UtmParams } from 'modules/utm'
 
-import { useAppCode } from '../hooks'
 import { appDataInfoAtom } from '../state/atoms'
 import { AppDataHooks, AppDataOrderClass } from '../types'
 import { buildAppData, BuildAppDataParams } from '../utils/buildAppData'
 import { getAppData } from '../utils/fullAppData'
 
 export type UseAppDataParams = {
+  appCodeWithWidgetMetadata: AppCodeWithWidgetMetadata | null
```

Comment:
As I commented in the `AppCodeWithWidgetMetadata` type definition, I think makes more sense to have all the 3 items (appCode, env, widget) as individual params

Reply:
related to the above, resolving